src/app/components/views/location-management-page/location-list-pane/location-list-pane.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="pane-wrapper shadow bg-white rounded login-for mb-3">
+    <h3 class="dynamic-title">{{ currentFormMode.title }}</h3>
+    <ng-template [ngIf]="locations !== undefined" [ngIfElse]="loadingIcon">
+        <div class="control-button-wrapper">
+            <button
+                *ngFor="let button of currentFormMode.buttons"
+                [ngClass]="button.classes"
+                [innerHTML]="button.title"
+                (click)="button.click()"
+            ></button>
+        </div>
+        <div [hidden]="!currentFormMode.displayLocationForm">
+            <form [formGroup]="locationForm" class="locationForm">
+                <div class="form-group">
+                    <input class="form-control" placeholder="Location Name" formControlName="name" />
+                </div>
+                <div class="row form-group">
+                    <div class="col-8 left-form-field">
+                        <input type="text" class="form-control" placeholder="Street Address" formControlName="streetNumber" />
+                    </div>
+                    <div class="col-4 right-form-field">
+                        <input type="text" class="form-control" placeholder="City" formControlName="city" />
+                    </div>
+                </div>
+                <div class="row form-group">
+                    <div class="col-4 left-form-field">
+                        <input type="text" class="form-control" placeholder="Prov./State" formControlName="provState" />
+                    </div>
+                    <div class="col-4 mid-form-field">
+                        <input type="text" class="form-control" placeholder="Postal Code" formControlName="postalCode"/>
+                    </div>
+                    <div class="col-4 right-form-field">
+                        <input type="text" class="form-control" placeholder="Country" formControlName="country" />
+                    </div>
+                </div>
+            </form>
+        </div>
+        <ng-template [ngIf]="locations.length > 0" [ngIfElse]="emptyList">
+            <hr style="margin: 0;">
+            <div class="pre-scrollable location-list" [ngClass]="currentFormMode.displayLocationForm ? 'form-enabled' : 'form-disabled'">
+                <app-location-list-item
+                    *ngFor="let locItem of locations"
+                    [location]="locItem"
+                    (click)="onSelectLocation(locItem)"
+                    (editButtonClick)="onLocationEditInitiate($event)"
+                    (deleteButtonClick)="onLocationDelete($event)"
+                ></app-location-list-item>
+            </div>
+        </ng-template>
+        <ng-template #emptyList>
```

Comment:
I'm not able to test this fully for the reasons stated in https://github.com/vitokhangnguyen/InvenTurbo-backend/pull/11#discussion_r396013891.

However, I can say that this `<ng-template>` tag is _not_ rendering in [Ikea](http://localhost:8080/api/organization/Ejp7mlGIx6cQ1j9ZNIWb)'s case; the spinners are just spinning forever.

I'll tentatively approve, assuming this works better for other organizations.

Reply:
I fixed the problem at https://github.com/vitokhangnguyen/InvenTurbo-backend/pull/11#discussion_r396013891 so this should work now.

On a side note, please be aware that my PR requires an extra dependency (Google Map API) so remember to do `npm i` before you test.

Test again when you have time!

Reply:
Resolved, thanks!